frontend/ui/src/pages/Inbox/Messenger/ConversationStatus/index.module.scss

```diff
@@ -0,0 +1,60 @@
+@import 'assets/scss/colors.scss';
+@import 'assets/scss/fonts.scss';
+
+.conversationStatus {
+  width: 96px;
+  height: 26px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  margin-left: 5px;
+  border-bottom: 1px var(--color-light-gray) solid;
+  @include font-base;
+  color: var(--color-text-contrast);
+}
+
+.conversationStatus .closedButtonWrapper {
+  padding: 6px 6px 6px 0;
+  text-align: center;
+  border-bottom: 3px transparent solid;
+}
+
+.conversationStatus .closedButton {
+  width: 100%;
+  color: var(--color-text-gray);
+  line-height: 16px;
+  cursor: pointer;
+}
+
+.conversationStatus.closed .closedButtonWrapper {
+  border-bottom: 5px var(--color-soft-green) solid;
+  text-align: center;
+}
+
+.conversationStatus.closed .closedButton {
+  font-weight: 700;
+  color: var(--color-text-contrast);
+}
+
+.conversationStatus .openButtonWrapper {
+  border-bottom: 3px transparent solid;
+  padding: 6px 6px 6px 0;
+  text-align: center;
+}
+
+.conversationStatus .openButton {
+  width: 100%;
+  color: var(--color-text-gray);
+  line-height: 16px;
+  cursor: pointer;
+}
+
+.conversationStatus.open .openButtonWrapper {
+  border-bottom: 5px var(--color-red-alert) solid;
+}
+
+.conversationStatus.open .openButton {
```

Comment:
Are you sure that you need to prefix all of them with `.conversationStatus`? Because this is a css module there should already be an automatic prefix

Reply:
true! this is totally unnecessary, will fix!